### ISSUE TITLE
Following 2 same-named communities on different homeservers results in weird ui behavior

### ISSUE THREAD

**Issue by stephen304** [krawieck/lemmur#276]
### Device info

- OS: Android
- OS version: Lineage 18.1
- Device: Pixel 2
- Lemmur version: 0.6.0

### Describe the bug

I followed both !cats@lemmy.ml and !cats@midwest.social. Both communities show up as !cats@midwest.social in the community list, and both go to !cats@midwest.social, and !cats@lemmy.ml doesn't show up at all but visiting it manually shows that I am following it.

### Steps to reproduce

1. Follow both !cats@lemmy.ml and !cats@midwest.social
2. Go to community list

### Relevant logs

  <summary>Logs</summary>
  
  Paste your logs here. Logs can be found in lemmur: settings > about lemmur > logs.
  
  I don't see an option for logs:
![image](https://user-images.githubusercontent.com/981915/139596289-c0d4a55a-8141-4e48-b36e-92111b7fb8b5.png)

### Expected behavior

Should show both communities

### Screenshots/Screencasts

![image](https://user-images.githubusercontent.com/981915/139596338-40970bb7-2809-426a-a169-0568c1e64034.png)

**Comment by shilangyu** [comment]
Are you doing it all from a single account? ie. Following both cats communities from the Lemmy instance? Where !cats@midwest.social is federated on lemmy.ml? 

Ps. Yes, there isn't a logs page yet, we added it to the issue template to get ready for the next release which includes a logs page

**Comment by stephen304** [comment]
@shilangyu Yes this is on a new account on Lemmy, I just searched a bunch of things to follow and cats happened to show both of those.

**Comment by shilangyu** [comment]
Alright, thank you for reporting. We will look into it
